dependencies.yaml

```diff
@@ -370,6 +371,26 @@ dependencies:
               - *libcusolver114
               - *libcusparse_dev114
               - *libcusparse114
+      - output_types: pyproject
+        matrices:
+          - matrix:
+              cuda: "12.*"
+            packages:
+              - nvidia-cublas-cu12
+              - nvidia-cufft-cu12
+              - nvidia-curand-cu12
+              - nvidia-cusparse-cu12
+              - nvidia-cusolver-cu12
+          - matrix:
+              cuda: "11.*"
+            packages:
+          - matrix:
+            packages:
+              - nvidia-cublas
```

Comment:
Why do these have no -cu11 suffixes?

Reply:
I thought we'd agreed that the default matrix should not have suffixes and list quasi-packages (since it will never actually be used) in order to convey to the user what the dependencies are. Am I mistaken? Should they have suffixes?

Reply:
My main question is what happens if users install the CUDA 11 wheel? The PyPI page makes it sound like they’ll get a warning. https://pypi.org/project/nvidia-cublas/

> A fake package to warn the user they are not installing the correct package.

Reply:
Indeed, if they install the CUDA 11 wheel they will get a fake package. However, this PR does not add any dependencies on CUDA 11 wheels, real or fake, so it shouldn't be an issue.

Reply:
Does the install log include warnings? Are they “loud”? That could be confusing to users if so.

Reply:
That's because the CUDA 11 matrix doesn't have any dependencies. The suffixless packages you see here are only for the default matrix entry, which never gets invoked.

Reply:
> That's because the CUDA 11 matrix doesn't have any dependencies. The suffixless packages you see here are only for the default matrix entry, which never gets invoked.

Ack. I misread the PR. I did not see that empty list, I thought CUDA 11 was using suffixless packages.